src/metabase/lib/js/metadata.cljs

```diff
@@ -365,15 +385,15 @@
   (some-> metadata :segments deref (get segment-id) deref))
 
 (defn- tables [metadata database-id]
-  (for [[_id table-delay] (some-> metadata :tables deref)
-        :let              [a-table (some-> table-delay deref)]
-        :when             (and a-table (= (:db-id a-table) database-id))]
+  (for [[_id table-delay]  (some-> metadata :tables deref)
+        :let               [a-table (some-> table-delay deref)]
+        :when              (and a-table (= (:db-id a-table) database-id))]
     a-table))
 
 (defn- fields [metadata table-id]
-  (for [[_id field-delay] (some-> metadata :fields deref)
-        :let              [a-field (some-> field-delay deref)]
-        :when             (and a-field (= (:table-id a-field) table-id))]
+  (for [[_id field-delay]  (some-> metadata :fields deref)
+        :let               [a-field (some-> field-delay deref)]
+        :when              (and a-field (= (:table-id a-field) table-id))]
```

Comment:
Are these changes needed?

Reply:
No, I'm not sure where those came from. Fixed.